runtime.md

```diff
@@ -11,8 +11,8 @@ The state of a container includes the following properties:
 
 * **`ociVersion`** (string, REQUIRED) is the OCI specification version used when creating the container.
 * **`id`** (string, REQUIRED) is the container's ID.
-This MUST be unique across all containers on this host.
-There is no requirement that it be unique across hosts.
+    This MUST be unique across all containers on this host.
+    There is no requirement that it be unique across hosts.
 * **`status`** (string, REQUIRED) is the runtime state of the container.
```

Comment:
I don't think this is a browser thing.  Looking at the rendered HTML:

```
$ curl -s https://github.com/Mashimiao/specs/blob/3d92dd2a8907af6fbabe47caf2f04e8cf628b34f/runtime.md | grep -A10 -B10 status | head -n21
</a><ul><a name="user-content-runtimeState">
<li>
<p><strong><code>ociVersion</code></strong> (string, REQUIRED) is the OCI specification version used when creating the container.</p>
</li>
<li>
<p><strong><code>id</code></strong> (string, REQUIRED) is the container's ID.
This MUST be unique across all containers on this host.
There is no requirement that it be unique across hosts.</p>
</li>
</a><li><a name="user-content-runtimeState">
<p><strong><code>status</code></strong> (string, REQUIRED) is the runtime state of the container.
The value MAY be one of:</p>
</a><ul><a name="user-content-runtimeState">
</a><li><a name="user-content-runtimeState"><code>creating</code>: the container is being created (step 2 in the </a><a href="#lifecycle">lifecycle</a>)</li>
<li><code>created</code>: the runtime has finished the <a href="#create">create operation</a> (after step 2 in the <a href="#lifecycle">lifecycle</a>), and the container process has neither exited nor executed the user-specified program</li>
<li><code>running</code>: the container process has executed the user-specified program but has not exited (after step 5 in the <a href="#lifecycle">lifecycle</a>)</li>
<li><code>stopped</code>: the container process has exited (step 7 in the <a href="#lifecycle">lifecycle</a>)</li>
</ul>
<p>Additional values MAY be defined by the runtime, however, they MUST be used to represent new runtime states not defined above.</p>
</li>
<li>
```

There are more `user-content-runtimeState` names than there should be, one of those `<a>` tags wraps the first two `<li>` entries, and only the `status` `<li>` entry gets some internal `<p>` tags.  It seems like GitHub may be having issues with the partially-[loose][1] list, but according to their GFM spec (and [this example][2]), a single item with a blank line between two block-level elements should be enough to make the *whole list* loose.

So I think this is a GitHub rendering bug, and we should just accept the broken rendering until we can get GitHub's Markdown renderer patched.  But it's probably worth washing this Markdown through Pandoc too to get a second opinion.

[1]: https://github.github.com/gfm/#loose
[2]: https://github.github.com/gfm/#example-284